src/Loader/CVRPModel.java

```diff
@@ -0,0 +1,54 @@
+package Loader;
+
+public class CVRPModel {
+    private int[][] coords;
+    private int[][] demands;
```

Comment:
Tutaj można rozważyć korzystanie z obiektowych Integer ów. Myślę, że może być łatwiej znaleźć wtedy jakieś błędy przy implementacji jak będzie nam strzelać nullem, niż 0 (z tego co wiem to Java przy tworzeniu new int[size] wypełnia ją 0)